Q3DC/Q3DC.py

```diff
@@ -1379,21 +1379,23 @@ def computeAngles(self, line1, line2, states):
         return result
 
     def computeLinePoint(self, lineA, lineB, point):
-        # Adapted from vtkLine::DistanceToLine(const double x[3], const double p1[3], const double p2[3])
-        pointLineA = point - lineA
-        lineALineB = lineA - lineB
+        if np.all(lineA == lineB):
```

Comment:
Should we use `np.allclose` with a tolerance ? 

See https://numpy.org/doc/stable/reference/generated/numpy.allclose.html

Reply:
I did not know about these methods. This seems like a good approach to prevent floating point errors